common/technology_sharing/eaw_tech_sharing_groups.txt

```diff
@@ -142,144 +121,116 @@ technology_sharing_group = {
-
+	
 	id = equestrian_research
 	name = equestrian_research_name
 	desc = equestrian_research_desc
 	picture = GFX_commonwealth_research
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	available = {
 		OR = {
-			is_in_faction_with = EQS
 			original_tag = EQS
+			is_in_faction_with = EQS
 		}
 	}
 }
 
 technology_sharing_group = {
-
+	
 	id = school_for_gifted_griffons
 	name = school_for_gifted_griffons_name
 	desc = WNG_school_for_gifted_griffons_desc
 	picture = GFX_technology_sharing_default
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	available = {
-		OR = {
-			tag = event_target:equestria
-			tag = WNG
-		}
-		OR = {
-			AND = {
-				event_target:equestria = { has_government = democratic }
-				WNG = { has_government = democratic }
-			}
-			AND = {
-				event_target:equestria = { has_government = communism }
-				WNG = { has_government = communism }
-			}
-			AND = {
-				event_target:equestria = { has_government = fascism }
-				WNG = { has_government = fascism }
-			}
-			AND = {
-				event_target:equestria = { has_government = neutrality }
-				WNG = { has_government = neutrality }
+		WNG = {
+			exists = yes
+			event_target:equestria = {
+				exists = yes
+				NOT = { has_war_with = WNG }
+				if = {
+					limit = { has_government = democratic }
+					ROOT = { has_government = democratic }
+				}
+				else_if = {
+					limit = { has_government = fascism }
+					ROOT = { has_government = fascism }
+				}
+				else_if = {
+					limit = { has_government = neutrality }
+					ROOT = { has_government = neutrality }
+				}
+				else = {
+					ROOT = { has_government = communism }
+				}
 			}
 		}
-		NOT = { WNG = { has_war_with = event_target:equestria } }
-		country_exists = event_target:equestria
-		country_exists = WNG
 	}
 }
 
-
 technology_sharing_group = {
+	
 	id = RIV_science_society_group
 	name = RIV_science_society_group_name
 	desc = RIV_science_society_group_desc
 	picture = GFX_technology_sharing_default
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	available = {
 		is_river_fed_member = yes
 	}
 }
 
 technology_sharing_group = {
+	
 	id = FAT_research_group
 	name = FAT_research_group_name
 	desc = FAT_research_group_desc
 	picture = GFX_technology_sharing_default
-
+	
 	research_sharing_per_country_bonus = 0.15
-
+	
 	available = {
-		FAT = { exists = yes is_subject = no }
-		OR = {
-			tag = FAT
+		if = {
+			limit = { tag = FAT }
+			is_subject = no
+		}
+		else = {
 			is_in_faction_with = FAT
 			is_subject_of = FAT
 		}
 	}
 }
 
 technology_sharing_group = {
-
+	
 	id = northern_science_organisation
 	name = northern_science_organisation_name
 	desc = northern_science_organisation_desc
 	picture = GFX_technology_sharing_default
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	available = {
+		GRF = { is_faction_leader = yes }
 		is_in_faction_with = GRF
-		GRF = {
-			is_faction_leader = yes
-		}
-	}
-}
-
-technology_sharing_group = {
-
-	id = CRY_SOL_magic_tech_share
```

Comment:
Why did you remove this?

Reply:
It was swapped, although I do not know why.